l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -1022,6 +1025,40 @@ def set_art73(self, FatturaBody, invoice_data):
         if FatturaBody.DatiGenerali.DatiGeneraliDocumento.Art73:
             invoice_data['art73'] = True
 
+    def set_roundings(self, FatturaBody, invoice):
+        Summary_datas = FatturaBody.DatiBeniServizi.DatiRiepilogo
```

Comment:
`data` è già plurale, e le variabili vanno minuscole (https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

Reply:
Ok! Rinomino